src/OpenColorIO/fileformats/FileFormatSpi3D.cpp

```diff
@@ -225,4 +231,3 @@ FileFormat * CreateFileFormatSpi3D()
 }
 
 } // namespace OCIO_NAMESPACE
-
```

Comment:
Please add a unit test in tests/cpu/fileformats/FileFormatSpi3D_tests.cpp.  See tests/data/files/comp2.spi3d for an example of the format.  You could use a LUT size of 3x3x3 to keep the example short (it only requires 27 entries).

For small LUTs, we sometimes like to put the LUT into the code rather than using an external file, but either way is OK.  There are lots of examples in tests/cpu/fileformats/FileFormatCTF_tests.cpp that show both ways of working (external file or embed in the code).

Reply:
I have added a unit test for the exception according to the format of other unit tests. Can you review it? Thanks!